scripts/images/rancher-images.txt

```diff
@@ -1,25 +1,35 @@
 docker.io/rancher/fleet-agent:v0.13.0
 docker.io/rancher/fleet:v0.13.0
+docker.io/rancher/kubectl:v1.29.2
 docker.io/rancher/kubectl:v1.33.1
 docker.io/rancher/mirrored-cluster-api-controller:v1.10.2
 docker.io/rancher/mirrored-fluent-fluent-bit:3.1.8
 docker.io/rancher/mirrored-fluent-fluent-bit:3.1.8-debug
-docker.io/rancher/mirrored-grafana-grafana:11.5.5
+docker.io/rancher/mirrored-grafana-grafana:11.1.0
+docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.4.3
 docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.1
 docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.2
 docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.3
```

Comment:
I can’t guarantee that the images below aren’t used elsewhere, so I’ve kept them for now. If it’s confirmed that they can be safely removed, I’m happy to do so, which would also help reduce the ISO size.
- docker.io/rancher/mirrored-kiwigrid-k8s-sidecar:1.30.0
- docker.io/rancher/mirrored-kube-state-metrics-kube-state-metrics:v2.15.0
- docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.1
- docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.2
- docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.3
- docker.io/rancher/mirrored-library-nginx:1.27.2-alpine

Reply:
refer https://github.com/harvester/harvester-installer/blob/d5d7ec5aa3be8e9c0933bc591b2f524c7a54521e/scripts/images/rancher-images.txt#L7

there are very few duplicated images, but not the case `...webhook-certgen`